CHANGELOG_iOS.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.79.124](https://github.com/brave/brave-browser/releases/tag/v1.79.124)
+
+ - Added search omnibox UI upgrades. ([#44257](https://github.com/brave/brave-browser/issues/44257))
+ - Added "Deepseek R1" model to Leo for premium users. ([#45454](https://github.com/brave/brave-browser/issues/45454﻿))
```

Comment:
Think there's a formatting issue. If you paste the above into a text editor, you'll see `https://github.com/brave/brave-browser/issues/45454<0xfeff>﻿`. Clicking on the link via https://github.com/brave/brave-browser/blob/4df7f18a4d5567da8eda152e56290ecdeed1c0d3/CHANGELOG_iOS.md doesn't work.

![image](https://github.com/user-attachments/assets/e8da8c3e-ccd8-450a-8641-0ca2afe93185)